paper-server/patches/sources/net/minecraft/server/level/ServerPlayer.java.patch

```diff
@@ -1,12 +1,13 @@
 --- a/net/minecraft/server/level/ServerPlayer.java
 +++ b/net/minecraft/server/level/ServerPlayer.java
-@@ -235,7 +_,8 @@
+@@ -235,7 +_,9 @@
      private int levitationStartTime;
      private boolean disconnected;
      private int requestedViewDistance = 2;
 -    public String language = "en_us";
 +    public String language = null; // Paper - default to null
 +    public java.util.Locale adventure$locale = java.util.Locale.US; // Paper
++    public java.util.Locale effectiveLocale = null; // Paper
```

Comment:
Ought to be annotated with `@Nullable`. I believe nms uses the javax annotation set for this type.